jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFComponentProcessor.java

```diff
@@ -294,7 +300,7 @@ private void processJSFArtifactsForInjection(final DeploymentUnit deploymentUnit
                                         if (child.getTag() == JsfTag.MANAGED_BEAN_CLASS) {
                                             installManagedBeanComponent(className, moduleDescription, module, deploymentUnit, applicationClassesDescription);
                                         } else {
-                                            installJsfArtifactComponent(child.getTag().getTagName(), className, isCDI, moduleDescription, module, deploymentUnit, applicationClassesDescription);
+                                            installJsfArtifactComponent(child.getTag().getTagName(), className, moduleDescription, module, deploymentUnit, applicationClassesDescription);
```

Comment:
Yep; isCDI isn't needed any longer. Any other jsf subsystem code that is conditionally checking for CDI can stop too once JSFResourceDefinition declare an 'org.wildfly.faces' capability that requires WELD_CAPABILITY_NAME.